src/main/java/io/pakland/mdas/githubstats/application/internal/AggregateReviews.java

```diff
@@ -2,18 +2,55 @@
 
 import io.pakland.mdas.githubstats.domain.entity.Review;
 import io.pakland.mdas.githubstats.domain.entity.ReviewAggregation;
+import io.pakland.mdas.githubstats.domain.entity.Team;
+import io.pakland.mdas.githubstats.domain.entity.User;
 import org.springframework.stereotype.Service;
 
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 @Service
 public class AggregateReviews {
 
     ReviewAggregation reviewAggregation = new ReviewAggregation();
     public AggregateReviews() {}
 
-    public ReviewAggregation execute(List<Review> reviews) {
-        return reviewAggregation.aggregate(reviews);
+    public Map<User, Map<Team, ReviewAggregation>> execute(List<Review> reviews) {
+        // {
+        //   "userA": {
+        //      "teamA": [reviewsFromUserAInTeamA],
+        //      "teamB": [reviewsFromUserAInTeamB]
+        //   },
+        //   "userB": { "teamA": [reviewsFromUserBInTeamA] },
+        //   "userC": { "teamB": [reviewsFromUserCInTeamB] } ...
+        // }
```

Comment:
We can keep this comments, though I would remove them or add mor explanation. Nonetheless, I really like the approach you have taken. However, how do you feel about keeping the hierarchy of `Organization` -> `Team` -> `User` and using such to build this data structure?